Automattic/pocket-casts-android
# Improve What's New stars animation

## Description

Improved animation for What's New rating. Timing between In and Out transitions might be a bit inaccurate. Please fix in the main PR if needed. Right now I use 2 seconds for gone time and 5 seconds for visible time.

## Testing Instructions

1. Apply [this patch](https://github.com/user-attachments/files/16496153/anim.patch).
2. Open the app.
3. Verify the animation.

## Screenshots or Screencast 

![screen-20240805-150228-ezgif com-crop](https://github.com/user-attachments/assets/2bafb707-18b9-4b2e-87ce-520b7ef6017e)

https://github.com/user-attachments/assets/bb5d2caa-067d-46bd-9da1-ee994f85dc87
